docs/client/basic/sections/commandline.md

```diff
@@ -17,6 +17,10 @@ Make a subdirectory called `<name>` and create a new Meteor app there.
 Serve the current app at [http://localhost:3000](http://localhost:3000)
 using Meteor's local development server.
 
+#### `meteor debug`
+
+Run the project, but suspend the server process for debugging.
```

Comment:
Worth adding a link to the advanced section here, and make the text something less cryptic like "Run the project with Node Inspector attached, so that you can step through your server code line by line."